rust/datafusion/src/execution/physical_plan/hash_aggregate.rs

```diff
@@ -500,50 +500,49 @@ impl RecordBatchReader for GroupedHashAggregateIterator {
                 )),
             };
             result_arrays.push(array.map_err(ExecutionError::into_arrow_external_error)?);
+        }
```

Comment:
If you look at this with a whitespace blind diff: https://github.com/apache/arrow/pull/7924/files?w=1 the change might be easier to see. The issue is that the current code adds the aggregate expressions one per *grouping expression* rather than once per operator. 

The change is in essence, to move the location of the closing `}` for aggregate expressions